react/HOWTO.md

```diff
@@ -0,0 +1,29 @@
+# How to integrate Cozy UI to your application
+## Add Cozy-UI as a dependency
+
+Use [`yarn`](https://yarnpkg.com/) to add Cozy UI to a dependency to your project.
+
+```bash
+$ yarn add cozy-ui
```

Comment:
I think this step should be in the README.md and should replace the stylus plugin section that no one uses.